components/alert/alert.css

```diff
@@ -10,22 +10,6 @@
   font-size: 1.75rem;
 }
```

Comment:
you can use px here for font size (small units). Writing units in decimal is not good practice.

Reply:
I thought we can use px if the units are less than 1 rem i.e 10 px in my case